devito/ops/transformer.py

```diff
@@ -221,10 +221,13 @@ def create_ops_fetch(f, name_to_ops_dat, time_upper_bound):
 def create_ops_par_loop(trees, ops_kernel, parameters, block, name_to_ops_dat,
                         accessible_origin, par_to_ops_stencil, dims):
     it_range = []
+    devito_to_ops_indexer = 1
```

Comment:
I think this entire piece of code is incredibly and uselessly complicated...

btw, what do you need the ``it_ranges`` for? can you provide examples?

Reply:
In OPS we define an ´OPS_Kernel_range´ variable which contains all the minimal and maximal values in each dimension. We currently are getting those from devito with `it_ranges`. So in a simple example where we add one to each point of a grid in each time step we would have:
`int OPS_Kernel_0_range[4] = {x_m, x_M + 1, y_m, y_M + 1}`